config.json

```diff
@@ -334,6 +334,14 @@
           "math"
         ]
       },
+      {
+        "slug": "perfect-numbers",
+        "name": "Perfect Numbers",
+        "uuid": "696b5088-ec86-4d60-8ac1-2cc1d89dee52",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
maybe more than 2

Reply:
2 is my default for an easy exercise but I’d be glad to tweak it. Did you have something in mind? 3 to mark it as a harder easy problem or 4 as a easier medium problem?

Reply:
Requiring an enum bumps it up for me. I hate quibbling over gradations of "easy", particularly when it's completely invisible on the site. But it's not a medium difficulty exercise.

Leave it as it then, not worth changing.